functests/common.sh

```diff
@@ -194,7 +194,8 @@ function uninstall_minio() {
     "${DIRECT_CSI_CLIENT}" volumes ls
 
     while true; do
-        count=$("${DIRECT_CSI_CLIENT}" volumes ls | grep -v WARNING| wc -l)
+        count=$("${DIRECT_CSI_CLIENT}" volumes ls | grep -vc WARNING)
```

Comment:
You should use ` | wc -l` otherwise `grep -vc` will return zero on failure.

Reply:
`grep | wc -l` leads to lint failure.

Reply:
> `grep | wc -l` leads to lint failure. 

We need add to ignore list